modules/social_features/social_email_broadcast/src/SocialEmailBroadcast.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\social_email_broadcast;
+
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Session\AccountInterface;
+
+/**
+ * Common service contains methods for "Social Email Broadcast".
+ */
+class SocialEmailBroadcast {
+
+  /**
+   * Immediate frequency.
+   */
+  const FREQUENCY_IMMEDIATELY = 'immediately';
+
+  /**
+   * None frequency.
+   */
+  const FREQUENCY_NONE = 'none';
+
+  /**
+   * Bulk emails user's frequencies table name.
+   */
+  const TABLE = 'user_email_send';
```

Comment:
Not sure if just `TABLE` isn't too broad name..
Should we rename it to something like `USER_FREQUENCY_TABLE` ?